test/e2e/network/network_policy.go

```diff
@@ -1785,9 +1791,180 @@ var _ = SIGDescribe("NetworkPolicy [LinuxOnly]", func() {
 	})
 })
 
+var _ = SIGDescribe("NetworkPolicy [Feature:SCTPConnectivity][LinuxOnly]", func() {
```

Comment:
According to https://github.com/kubernetes/enhancements/blob/master/keps/sig-network/0015-20180614-SCTP-support.md#sctp-connectivity-tests these should be marked as `[Disruptive]`?

Reply:
Yes. And we should belatedly fix the other SCTPConnectivity tests to have that now too

Reply:
Done, and added a separate commit to mark the old ones disruptive as well.